AppCenterCrashes/AppCenterCrashes/Internals/MSCrashesInternal.h

```diff
@@ -61,17 +61,29 @@ static NSString *const kMSAppDidReceiveMemoryWarningKey = @"MSAppDidReceiveMemor
  */
 - (void)configureCrashReporterWithUncaughtExceptionHandlerEnabled:(BOOL)enableUncaughtExceptionHandler;
 
-/*
+/**
  * Track handled exception directly as model form.
- * This API is not public and is used by wrapper SDKs.
+ * This API is used by wrapper SDKs.
  *
  * @param exception model form exception.
  * @param properties dictionary of properties.
  * @param attachments a list of attachments.
+ *
+ * @return handled error ID.
+ */
+- (NSString *)trackModelException:(MSException *)exception
+                   withProperties:(nullable NSDictionary<NSString *, NSString *> *)properties
+                  withAttachments:(nullable NSArray<MSErrorAttachmentLog *> *)attachments;
+
+/**
+ * Get a generic error report representation for an handled exception.
+ * This API is used by wrapper SDKs.
+ *
+ * @param errorID handled error ID.
+ *
+ * @return an error report.
  */
-+ (void)trackModelException:(MSException *)exception
-             withProperties:(nullable NSDictionary<NSString *, NSString *> *)properties
-            withAttachments:(nullable NSArray<MSErrorAttachmentLog *> *)attachments;
+- (MSErrorReport *)buildHandledErrorReportWithErrorID:(NSString *)errorID;
```

Comment:
I think our convention for apple sdks is to use `Id`rather than `ID` (for method name and parameter)

Reply:
The "up-to-date" naming convention by Apple is to use `ID` instead `Id` and we adopted that convention a while ago. There are going to be inconsistencies between `ID` and `Id`, but we should follow this pattern in this repo going forward.

@clpolet Feel free to comment on this convention.

Reply:
I am wrong, ignore me!